packages/plugin-utils/src/processing/registerFillProcessingComponents.ts

```diff
@@ -91,10 +183,97 @@ export default function registerFillProcessingComponents(
     }
   );
 
+  const buttonComponents: {
+    componentIds: string[];
+    label: string;
+    variant: 'plain' | 'regular';
+  }[] = [
+    {
+      componentIds: inspectorBarComponentIds,
+      variant: 'plain',
+      label: inspectorBarLabel
+    },
+    {
+      componentIds: navigationBarComponentIds,
+      variant: 'regular',
+      label: navigationBarLabel
+    },
+    {
+      componentIds: canvasBarComponentIds,
+      variant: 'regular',
+      label: canvasBarLabel
+    },
+    {
+      componentIds: canvasMenuComponentIds,
+      variant: 'plain',
+      label: canvasMenuLabel
+    }
+  ];
+
+  buttonComponents.forEach(({ componentIds, label, variant }) => {
+    const componentId = componentIds[0];
```

Comment:
Same here, seems weird that we have a list but always only consider the first entry.

Reply:
In my debugging frenzy, I think I missed rolling back the registering to all ids. Fixed in d56055a